src/it/rpm-dirmode/verify.groovy

```diff
@@ -1,3 +1,6 @@
+// Disable this test for now
+System.exit(0)
```

Comment:
Don't do it in this way.

You can add `pomExcludes` to m-invoker-p or create a file: `selector.groovy` with body 
```
return false
```

https://maven.apache.org/plugins/maven-invoker-plugin/run-mojo.html#pomExcludes
https://maven.apache.org/plugins/maven-invoker-plugin/examples/selector-scripts.html